packages/create-vite/src/index.ts

```diff
@@ -172,8 +172,7 @@ const FRAMEWORKS: Framework[] = [
         name: 'redwoodsdk-standard',
```

Comment:
Maybe we should rename this as `custom-redwoodsdk` or `custom-rwsdk` in the future to match the other names.